src/Microsoft.OData.Core/Parameterized.Microsoft.OData.Core.cs

```diff
@@ -1801,13 +1801,27 @@ internal static class Strings {
             return Microsoft.OData.TextRes.GetString(Microsoft.OData.TextRes.ReaderValidationUtils_NullValueForNonNullableType, p0);
         }
 
+        /// <summary>
+        /// A string like "A null value was found with the expected type '{0}[Nullable=True]'. The expected type '{0}[Nullable=True]' does not allow null values."
+        /// </summary>
+        internal static string ReaderValidationUtils_NullValueForNullableType(object p0) {
+            return Microsoft.OData.TextRes.GetString(Microsoft.OData.TextRes.ReaderValidationUtils_NullValueForNullableType, p0);
+        }
+

```

Comment:
should rebase to the master, because this file has been changed using different format